src/store/AppStore.js

```diff
@@ -19,11 +19,40 @@ const AppStore = types.model('AppStore', {
   image: types.optional(ImageStore, () => ImageStore.create({})),
   groups: types.optional(GroupsStore, () => GroupsStore.create({})),
   subjects: types.optional(SubjectStore, () => SubjectStore.create({})),
-  initialised: types.optional(types.boolean, false),
+  initialized: types.optional(types.boolean, false),
   projects: types.optional(ProjectsStore, () => ProjectsStore.create({})),
   modal: types.optional(ModalStore, () => ModalStore.create({})),
   transcriptions: types.optional(TranscriptionsStore, () => TranscriptionsStore.create({})),
   workflows: types.optional(WorkflowsStore, () => WorkflowsStore.create({})),
-})
+}).actions(self => ({
+  getResources: flow (function * getResources(params) {
```

Comment:
Noice, a super function that fetches resources in order of dependencies. I like me a super function.

However, I haven't had the chance to really try to break this function. e.g. is it possible to get `self.projects.reset()` to reset _without_ resetting the dependent workflows? Will there be a problem if I chuck in `project = 0`, which will evaluate to `false` in `!!params.project` ? etc etc etc

But as far as _normal functionality_ looks though, this is a good ol' 👍